examples/Polynomial_Optimization.ipynb

```diff
@@ -93,51 +52,104 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "******************************************************************************\n",
+      "This program contains Ipopt, a library for large-scale nonlinear optimization.\n",
+      " Ipopt is released as open source code under the Eclipse Public License (EPL).\n",
+      "         For more information visit http://projects.coin-or.org/Ipopt\n",
+      "******************************************************************************\n",
+      "\n",
+      "termination_status(model) = LOCALLY_SOLVED::TerminationStatusCode = 4\n",
+      "value(a) = 0.4999999966705758\n",
+      "value(b) = 0.49999999667062867\n",
+      "objective_value(model) = 0.24999999500590342\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "0.24999999500590342"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "using JuMP\n",
     "using Ipopt\n",
-    "m = Model(optimizer=IpoptOptimizer(print_level=0))\n",
-    "@variable m a >= 0\n",
-    "@variable m b >= 0\n",
-    "@constraint m a + b >= 1\n",
-    "@NLobjective(m, Min, a^3 - a^2 + 2a*b - b^2 + b^3)\n",
-    "JuMP.optimize(m)\n",
-    "@show JuMP.terminationstatus(m)\n",
-    "@show JuMP.resultvalue(a)\n",
-    "@show JuMP.resultvalue(b)\n",
-    "@show JuMP.objectivevalue(m);"
+    "model = Model(with_optimizer(Ipopt.Optimizer, print_level=0))\n",
+    "@variable(model, a >= 0)\n",
+    "@variable(model, b >= 0)\n",
+    "@constraint(model, a + b >= 1)\n",
+    "@NLobjective(model, Min, a^3 - a^2 + 2a*b - b^2 + b^3)\n",
+    "optimize!(model)\n",
+    "\n",
+    "@show termination_status(model)\n",
+    "@show value(a)\n",
+    "@show value(b)\n",
+    "@show objective_value(model)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "With the following equivalent model, [Ipopt](https://github.com/JuliaOpt/Ipopt.jl/) finds the correct optimal solution. The reason (although counter intuitive) of the difference is that with registered functions, only first order derivatives are available."
+    "Even with the following equivalent model model(containing a registered function), [Ipopt](https://github.com/JuliaOpt/Ipopt.jl/) is unable to find the correct optimal solution."
```

Comment:
missing space after "model"

Reply:
"Even" is not an appropriate link word, why should this one give a better solution ?
I would rather say "Note that the problem can be written equivalently as follows using [registered functions](http://www.juliaopt.org/JuMP.jl/v0.19.0/nlp/#User-defined-Functions-1)"

Reply:
True, "Even" is not appropriate unless we specify why registered functions can help. I have made the change.